### PR TITLE
Add `ContinuousColor` and `OrdinalColor` columns

### LINKED ISSUE #83: Native continuous colorscales/colormaps
## Elevator Pitch
It would be extremely helpful if `ipyforcegraph` supported continuous colormaps (e.g. `viridis`) as part of the Nunjucks templating. Users would be able to specify columns that contain continuous values (age, weight, etc.) and a colormap, and `ipyforcegraph` would be able to apply the color as part of the e.g. NodeShape behavior without requiring additional dependencies i.e. `matplotlib`. 

## Motivation
Bringing in additional dependencies to support styling with `ipyforcegraph` can complicate build chains, recipes, etc. It would be valuable to reduce complexity of leveraging ipyforcegraph to include common colormaps for continuous values, which are more complicated to implement than discrete maps (which can be easily added with e.g. dictionaries). 

## Design Ideas

Something akin to `{{ node.value | colormap('viridis') }}` would be extremely powerful.

### PR DESCRIPTION
**Pull request by nrbgt**
## Checklist

- [x] the PR target is correct
  - `main` for releases, docs-only fixes and post-release hot-fixes
  - `dev` for everything else
- [x] ran `doit lint` locally

## References

- fixes #83
- blocked by #68

## Code changes

- [x] adds `OrdinalColor` and `ContinuousColor`
  - [x] each has a `Scheme` enum with compatible types
  - [x] ordinal can be overloaded with custom colors in `range` (not really documented) 
- [x] unit tests
- [x] robot tests
- [x] docs

## User-facing changes

- users can now add relatively flexible color scales based on column values
  - `domain` values have to be known up-front, generally

### Screens

| note | shot |
|-|-|
| manual | image
| continuous and ordinal | image
| strawman | image

## Backwards-incompatible changes

- n/a

### CODE REVIEW

File: examples/_index.ipynb [hunk 1]
```diff
@@ -259,6 +259,15 @@
...
     "- [🏃 Test Behaviors](./Test_Behaviors.ipynb)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Future\n",
+    "\n",
+    "- [Scales](./Scales.ipynb)"
```
Comment (sanbales) [inline examples/_index.ipynb:269]: 🎨?

Comment (nrbgt) [reply examples/_index.ipynb:269]: ⚖️?

**Comment by nrbgt** [comment]
Going to relock here, then dig into tests. Ideally, we'd also hit the categorical ones as well.

**Review by sanbales — APPROVED** [review]
Working great!  No real concerns, only one minor nitpick comment which I'd be happy to have it ignored.

File: src/ipyforcegraph/behaviors/scales.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        (0.0, 1.0),
+        help=("the ``[min, max]`` to map to the scale's colors"),
+    ).tag(sync=True)
+
+    @T.validate("scheme")
+    def _validate_scheme(self, proposal: T.Bunch) -> Any:
+        return validate_enum(proposal, ContinuousColor.Scheme)
+
+
+class OrdinalColor(Column):
```
[the remark this answers is not in the document]
Comment (nrbgt) [reply src/ipyforcegraph/behaviors/scales.py:83]: 558a12e adds a manual scale example

**Pull request merged**